packages/elements/src/calendar/index.ts

```diff
@@ -1030,16 +1025,9 @@ export class Calendar extends ControlElement implements MultiValue {
    */
   private viewFormattedDate (segment: DateSegment, includeMonth = false): TemplateResult {
     const year = segment.year;
-    const isBC = year <= 0;
-    const includeEra = isBC;
+    const includeEra = year <= 0;
```

Comment:
I feel that previous code is a bit more clear when specifying `isBC`. What do you think ?

Reply:
I think the TranslateDirective will receive the variable name `includeEra` anyway.

how about this?
```suggestion
    const isBC = year <= 0;
```

  and return  with option `{ date, includeMonth, includeEra: isBC }`

Reply:
I think it's better than before. Looks good.